CONTRIBUTING.md

```diff
@@ -15,10 +15,10 @@ In order to contribute to this project, follow a few easy steps:
 1. [Fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) this repository and clone it on your machine
 2. Open the local repository with [Visual Studio Code](https://code.visualstudio.com) with the remote development feature enabled (install the [Remote Development extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack))
 3. Create a branch `my-awesome-feature` and commit to it
-4. Run `npm run lint`, `npm run test` and `npm run build` and verify that they complete without errors
-5. Push `my-awesome-feature` branch to GitHub and open a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+4. Run `npm run format`, `npm run lint`, `npm run test` and `npm run build`,  
+   or `npm run format && npm run lint && npm run test && npm run build` and verify that they complete without errors5. Push `my-awesome-feature` branch to GitHub and open a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
 6. Liked some of my work? Buy me a ☕ (or more likely 🍺)
 
 <p align="center">
   <a href="https://paypal.me/marcopolichetti" target="_blank"><img src="https://img.shields.io/badge/Donate-PayPal-ff3f59.svg"/></a>
-</p>
+</p>
```

Comment:
What's the difference here? Maybe LF?

Reply:
Yes, i managed to reverte it by overwriting it with the original source.

Reply:
Ok then fix, squash all commits and I'll merge! 🥳

Reply:
Perfect, i squashed it al now

Reply:
Thanks, it seems all fine except line 24 of contributing... still LF?

Reply:
You are right, sory for the inconvinience.
I fixed it now.